### PR TITLE
feat: supporting the 16.1.0 release

### PR DESCRIPTION
**Pull request by giacomoch**
For: https://github.ibm.com/cp4i/singularity-planning/issues/767

Testing: https://hyc-cip-jenkins.swg-devops.com/job/Automation/job/global_catalog_automation/386

All new operators work:

image

New instances work:

image

### CODE REVIEW

File: products/bash/find-catalog-sources/find-catalog-images.sh [hunk 1]
```diff
@@ -4,16 +4,16 @@ SCRIPT_DIR=$(dirname $0)
 
 # Data copied from the table in https://www.ibm.com/docs/en/cloud-paks/cp-integration/2023.4?topic=images-adding-catalog-sources-cluster
 # + the row from https://ibm-cloud.slack.com/archives/CT320FAH4/p1686903900324049?thread_ts=1686887428.360699&cid=CT320FAH4
```
Comment (DanRoseus) [inline products/bash/find-catalog-sources/find-catalog-images.sh:6]: I guess these are out of date...

**Review by DanRoseus — COMMENTED** [review]
Looks good, 1 minor comment on a comment. Before merging we need to make sure a service branch is created for the previous release.

File: products/bash/find-catalog-sources/find-catalog-images.sh (hunk 1 above)
Comment (giacomoch) [inline products/bash/find-catalog-sources/find-catalog-images.sh:6]:
```suggestion
# Data copied from the table in https://www.ibm.com/docs/en/cloud-paks/cp-integration/16.1.0?topic=images-adding-catalog-sources-cluster
```

Comment (giacomoch) [reply products/bash/find-catalog-sources/find-catalog-images.sh]: Fixed

**Review by laurensks — APPROVED** [review]
looks good to me

**Pull request merged**